src/composed/signed_key/mod.rs

```diff
@@ -87,6 +88,7 @@
 //!         packet::Subpacket::regular(packet::SubpacketData::Issuer(signing_key.key_id())),
 //!     ],
 //!     vec![],
+//!     SignatureVersionSpecific::V4 {}
```

Comment:
this is unfortunate

Reply:
I think this has been conclusively resolved by our iterative changes to Builder/SignatureConfig creation